client/src/static/mockedDataDemo.ts

```diff
@@ -11,6 +11,8 @@ export const staticFileURLs = [
 
 // eslint-disable-next-line
 export const buildInitialModelsList = ({ SIR_PN, SIR_FN }: any): Model.Model[] => {
+  const PN = GroMEt2Graph.parseGromet(SIR_PN);
```

Comment:
These names won't scale if we add more models, right? We will have different `PN` and `FN` for several models so we will need more specific names.

Reply:
True, this is just for now. Once we add more models etc. we can revise the code.